dm/master/server.go

```diff
@@ -1747,7 +1747,7 @@ func (s *Server) OperateMysqlWorker(ctx context.Context, req *pb.MysqlWorkerRequ
 		if resp, err = w.OperateMysqlWorker(ctx, req, s.cfg.RPCTimeout); err != nil {
 			return makeMysqlWorkerResponse(err)
 		}
-	} else {
+	} else if req.Op == pb.WorkerOp_StopWorker {
```

Comment:
How about unifying to use `switch .. case`?

Reply:
update in 0b80cf4